pglookout/pglookout.py

```diff
@@ -154,6 +154,15 @@ def load_config(self, _signal=None, _frame=None):
         self.replication_lag_failover_timeout = self.config.get("max_failover_replication_time_lag", 120.0)
         self.replication_catchup_timeout = self.config.get("replication_catchup_timeout", 300.0)
         self.missing_master_from_config_timeout = self.config.get("missing_master_from_config_timeout", 15.0)
+
+        if self.replication_lag_warning_boundary >= self.replication_lag_failover_timeout:
+            msg = "Replication lag warning boundary (%s) is not lower than its failover timeout (%s)"
+            msg = msg % (self.replication_lag_warning_boundary, self.replication_lag_failover_timeout)
+            self.log.warning(msg)
+            warning_boundary = self.replication_lag_failover_timeout // 2
```

Comment:
I'd perhaps just set warning to one less than failover limit in this case (or failover limit itself if result would be <= 0). If we want to enforce it's at most half then we should change the logic a bit so that we also update the warning threshold when settings are, say, 40 seconds warning and 60 seconds critical but I don't think that's something we want to do.

Reply:
the reason for half is that there is enough time for "outside" observer to react in case there is a warning, 
How about we set it to min(failover_timeout, 30) since 30 is the default ?

Reply:
Not sure I understand why the min is needed there. If failover timeout is less than warning timeout then just unconditionally set to failover timeout?

Reply:
Ah yes, now I understand the behaviour correctly. 
With the second commit in place, it doesnot matter if warning is necessarily lower. Because there is always force check in between.